### ISSUE TITLE
npm install mixes up architectures (OS X)

### ISSUE THREAD

**Issue by markandrus** [node-webrtc/node-webrtc#92]
Following up on #50, doing

```
git clone https://github.com/js-platform/node-webrtc.git
cd node-webrtc
npm install
```

on OS X 10.9 results in building WebRTC for i386 and `webrtc.node` for x86_64.

```
$ lipo -info third_party/libwebrtc/trunk/out/Release/libjingle_peerconnection.a
input file third_party/libwebrtc/trunk/out/Release/libjingle_peerconnection.a
is not a fat file
Non-fat file: third_party/libwebrtc/trunk/out/Release/libjingle_peerconnection.a
is architecture: i386
```

```
$ lipo -info build/Release/webrtc.node
Non-fat file: build/Release/webrtc.node is architecture: x86_64
```

Tried running `npm install --arch=x86_64`, but this strangely causes `webrtc.node` to be built for i386.

**Comment by modeswitch** [comment]
I've done no work to make sure that node-webrtc builds and runs on OSX yet. I'll do that for the next milestone, but for now my focus is on releasing a stable version for linux only.

Your issue may be related to this: https://code.google.com/p/libjingle/issues/detail?id=395

Maybe you can try building a 32-bit version of webrtc.node on your system to see if that works?

**Comment by markandrus** [comment]
Here's the fix that worked for me. Haven't determined how best to include it in the build script yet.

```
$ export GYP_DEFINES="host_arch=x64 target_arch=x64"
$ npm install
...
$ lipo -info build/Release/webrtc.node
Non-fat file: build/Release/webrtc.node is architecture: x86_64
```

**Comment by markandrus** [comment]
Resolved in #98
